form.schema.json

```diff
@@ -0,0 +1,217 @@
+        },
+        "sections": {
+          "type": "array",
+          "description": "An array of all the sections of the page. Sections are used to group related questions together.",
+          "properties": {
+            "isExpanded": {
+              "oneOf": [{ "type": "boolean" }, { "type": "string" }],
+              "description": "Either a string or a boolean. \nIf a string value, 'true' means this section should always be rendered in an expanded state, whereas 'false' means it should be rendered in a collapsed state. \nIf a boolean value, true means this section should always be rendered in an expanded state, whereas false means it should be rendered in a collapsed state."
+            },
+            "label": {
+              "type": "string",
+              "description": "The label of the section"
+            },
+            "questions": {
+              "type": "array",
+              "description": "An array of all the questions in a section. Questions make up the fields of the form",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "Unique identifier of the question. Should be unique across all questions in the form. Prefer using camelCase for this property."
+                },
+                "label": {
+                  "type": "string",
+                  "description": "The question label that is displayed to the user"
+                },
+                "required": {
+                  "type": "boolean",
+                  "description": "Determines whether the question is a required field or not. Required fields must be answered before the form can be submitted. They may also get marked with a red asterisk (*) in the UI to denote them as required fields.",
+                  "default": false
+                },
+                "questionOptions": {
+                  "type": "object",
+                  "description": "Specifies metadata relating the question. These metadata are used to determine how the question is rendered in the UI, and how it is validated among other things.",
+                  "properties": {
+                    "type": {
+                      "type": "string",
+                      "description": "Determines the datatype that the question accepts. The value of this property should be one of the supported datatypes."
+                    },
+                    "rendering": {
+                      "type": "string",
+                      "description": "Determines how the question is rendered in the UI. The value of this property should be one of the supported field types."
+                    },
+                    "concept": {
+                      "type": "string",
+                      "description": "The concept UUID of the backing concept for the question. This value should be a valid UUID of a concept that exists in the concept dictionary of your distribution."
```

Comment:
"The backing concept, referenced by UUID or concept source reference/mapping pair in the format CIEL:1234"